test/alternator/test_provisioned_throughput.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2024-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+# Test for ProvisionedThroughput
+# ProvisionedThroughput is part of a table definition
+# The following tests make sure we can get, set and update its value
+
+import pytest
+from botocore.exceptions import ClientError
+from decimal import Decimal
+from test.alternator.util import random_string, random_bytes, new_test_table
+
+def test_create_table(dynamodb):
```

Comment:
Please consider if you really want or need a separate test file just for this one test function. The existing test_table.py already has several tests involving billing mode, and how it works in createtable, describetable, and so on.

Reply:
Please add a comment describing what the raison d'etre of the test - what you wrote it for. I assume it's to verify that if BillingMode is set to provisioned, this setting and the provisioned throughput set during CreateTable is saved and returned by DescribeTable

Reply:
I assumed I'll add more tests with the additional functionality, please let me know if you prefer it all in an existing  file and I'll move it.

Reply:
I don't think this test framework will be great for tests that involve the actual **performance** of throughput provisioning (e.g., that if rcu=10 you only get 10 reads per second), so I don't expect you to be able to add a lot of additional tests to this file. But I'm not strongly opposed - there is no real penalty for having more files (it's not like the C++ tests where each additional source file adds a gigabyte and a whole day of compilation ;-)). So you can leave it a separate file.